diff --git a/userspace/libsinsp/cri.proto b/userspace/libsinsp/cri.proto
index 6e060c63..a7b5c694 100644
--- a/userspace/libsinsp/cri.proto
+++ b/userspace/libsinsp/cri.proto
@@ -727,7 +727,7 @@ message ContainerConfig {
     // use-cases (e.g. debugging).
     // TODO: Determine if we need to continue supporting these fields that are
     // part of Kubernetes's Container Spec.
-    bool stdin = 12;
+    bool _stdin = 12;
     bool stdin_once = 13;
     bool tty = 14;
 
@@ -919,9 +919,9 @@ message ExecSyncRequest {
 
 message ExecSyncResponse {
     // Captured command stdout output.
-    bytes stdout = 1;
+    bytes _stdout = 1;
     // Captured command stderr output.
-    bytes stderr = 2;
+    bytes _stderr = 2;
     // Exit code the command finished with. Default: 0 (success).
     int32 exit_code = 3;
 }
@@ -935,16 +935,16 @@ message ExecRequest {
     bool tty = 3;
     // Whether to stream stdin.
     // One of `stdin`, `stdout`, and `stderr` MUST be true.
-    bool stdin = 4;
+    bool _stdin = 4;
     // Whether to stream stdout.
     // One of `stdin`, `stdout`, and `stderr` MUST be true.
-    bool stdout = 5;
+    bool _stdout = 5;
     // Whether to stream stderr.
     // One of `stdin`, `stdout`, and `stderr` MUST be true.
     // If `tty` is true, `stderr` MUST be false. Multiplexing is not supported
     // in this case. The output of stdout and stderr will be combined to a
     // single stream.
-    bool stderr = 6;
+    bool _stderr = 6;
 }
 
 message ExecResponse {
@@ -957,19 +957,19 @@ message AttachRequest {
     string container_id = 1;
     // Whether to stream stdin.
     // One of `stdin`, `stdout`, and `stderr` MUST be true.
-    bool stdin = 2;
+    bool _stdin = 2;
     // Whether the process being attached is running in a TTY.
     // This must match the TTY setting in the ContainerConfig.
     bool tty = 3;
     // Whether to stream stdout.
     // One of `stdin`, `stdout`, and `stderr` MUST be true.
-    bool stdout = 4;
+    bool _stdout = 4;
     // Whether to stream stderr.
     // One of `stdin`, `stdout`, and `stderr` MUST be true.
     // If `tty` is true, `stderr` MUST be false. Multiplexing is not supported
     // in this case. The output of stdout and stderr will be combined to a
     // single stream.
-    bool stderr = 5;
+    bool _stderr = 5;
 }
 
 message AttachResponse {
